de.dentrassi.pm.aspect.mvnosgi/src/de/dentrassi/pm/aspect/mvnosgi/internal/Pom.java

```diff
@@ -0,0 +1,43 @@
+package de.dentrassi.pm.aspect.mvnosgi.internal;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+public class Pom
+{
+    private String groupId;
+
+    private String artifactId;
+
+    private String version;
+
+    public Pom ( String groupId, String artifactId, String version )
+    {
+        this.groupId = groupId;
+        this.artifactId = artifactId;
+        this.version = version;
+    }
+
+    @Override
+    public String toString ()
+    {
+        return generatePom ();
+    }
+
+    private String generatePom ()
+    {
+        final StringBuilder sb = new StringBuilder ( 1024 );
+        sb.append ( "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" );
+        sb.append ( "<project xmlns=\"http://maven.apache.org/POM/4.0.0\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\">" );
+        sb.append ( String.format ( "<modelVersion>4.0.0</modelVersion><groupId>%s</groupId><artifactId>%s</artifactId><version>%s</version>", groupId, artifactId, version ) );
+        sb.append ( "</project>" );
+        return sb.toString ();
```

Comment:
Is there already some code to generate a POM? A simple pom file builder or something similar?